StickyTabBarViewController/Classes/MainTabBarController.swift

```diff
@@ -28,7 +29,10 @@ open class StickyViewControllerSupportingTabBarController: UITabBarController, S
     
     /// Prepares View Controller to be embedded as a child (wrapped in another internal View Controller)
     /// - Parameter childViewController: A View Controller conforming to `Expandable` protocol
-    final public func configureCollapsableChild(_ childViewController: Expandable) {
+    /// - Parameter isFullScreenOnFirstAppearance: A boolean to determine if child view controller should be presented
+    /// full screen on first configuration
+    final public func configureCollapsableChild(_ childViewController: Expandable,
+                                                isFullScreenOnFirstAppearance: Bool) {
```

Comment:
I think this is a good example for default's value usage. 
Would the caller expect to have this on/off if this had a default value? Does the signature is clear enough to go down that road? 

I'm bringing it up as this breaks the client code & require a major bump version if this were already used after a beta period. I don't think you need to do anything here but I wanted to point out this scenario as we discussed default values in the past 👍

Reply:
I think this can be a good point of using default values, and also using it would prevent from a major bump as you mentioned. But I'm not sure if what would consumers expect from such a method. I can think of use cases for both having it collapsed or expanded on first setup, but cannot estimate which one would be used more often though. 😄I will just update README for now.

Reply:
I will also increase version to 0.0.4

Reply:
> But I'm not sure if what would consumers expect from such a method. I can think of use cases for both having it collapsed or expanded on first setup, but cannot estimate which one would be used more often though

Exactly, this is the nice thing about libraries. We gotta be extra careful & thoughtful of all different scenarios. Another way to circumvent this would be adding another method that does exactly the same but with the extra parameter, so the major number is not bumped but rather marked as a an API addition, thus no breaking changes in the client side will be introduced.

Reply:
Yes this would be a nice thing to do. 👍